core/dungeon-masters-guide/races/race-aasimar.xml

```diff
@@ -0,0 +1,85 @@
+			<p class="indent"><strong><em>Celestial Resistance.</em></strong> You have resistance to necrotic damage and radiant damage.</p>
+			<p class="indent"><strong><em>Celestial Legacy.</em></strong> You know the <i>light</i> cantrip. Once you reach 3rd level, you can cast the <i>lesser restoration</i> spell once with this trait, and you regain the ability to do so when you finish a long rest. Once you reach 5th level, you can cast the <i>daylight</i> spell once with this trait as a 3rd-level spell, and you regain the ability to do so when you finish a long rest. Charisma is your spellcasting ability for these spells.</p>
+			<p class="indent"><strong><em>Languages.</em></strong> You can speak, read, and write Common and Celestial.</p>      
+			<div class="reference">
+				<div element="ID_PHB_SPELL_LIGHT" />
+			</div> 
+			<div class="reference">
+				<div element="ID_PHB_SPELL_LESSER_RESTORATION" />
+			</div> 
+			<div class="reference">
+				<div element="ID_PHB_SPELL_DAYLIGHT" />
+			</div> 
+		</description>
+		<sheet display="false"/>
+		<rules>
+			<stat name="charisma" value="2" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<stat name="wisdom" value="1" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +2 (Aasimar)" supports="Custom Ability Score Increase 2" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +1 (Aasimar)" supports="Custom Ability Score Increase 1" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<grant type="Size" id="ID_SIZE_MEDIUM" />
+			<grant type="Vision" id="ID_VISION_DARKVISION" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<grant type="Language" id="ID_LANGUAGE_CELESTIAL" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<grant type="Racial Trait" id="ID_WOTC_DMG_RACIAL_TRAIT_AASIMAR_CELESTIAL_RESISTANCE" />
+			<grant type="Racial Trait" id="ID_WOTC_DMG_RACIAL_TRAIT_AASIMAR_CELESTIAL_LEGACY" />
+		</rules>
+	</element>
+
+	<element name="Celestial Resistance" type="Racial Trait" source="Dungeon Master’s Guide" id="ID_WOTC_DMG_RACIAL_TRAIT_AASIMAR_CELESTIAL_RESISTANCE">
+		<description>
+			<p>You have resistance to necrotic damage and radiant damage.</p>
+		</description>
+		<sheet>
+			<description>You have resistance to necrotic damage and radiant damage.</description>
+		</sheet>
```

Comment:
I think there's no reason for sheet description, since app displays resistances on the sheet.
```suggestion
		<sheet display="false" />
```